pay-api/src/pay_api/models/refunds_partial.py

```diff
@@ -50,6 +52,19 @@ class RefundsPartial(Audit, VersionedModel):  # pylint: disable=too-many-instanc
     refund_amount = db.Column(db.Numeric(19, 2), nullable=False)
     refund_type = db.Column(db.String(50), nullable=True)
 
+    @classmethod
+    def find_by_invoice_id(cls, invoice_id: int):
+        """Return refund partials by invoice id."""
+        return db.session.query(RefundsPartial)\
+            .join(PaymentLineItem, PaymentLineItem.id == RefundsPartial.payment_line_item_id)\
+            .filter(PaymentLineItem.invoice_id == invoice_id).all()
```

Comment:
Probably better to put this in the service and keep the models barebones?

You reference to payment_line_item in the model, which could cause some pain of circular dependencies in the future

Reply:
I wonder if there is a way to reference to PaymentLineItem, without having to include it at the top somehow

Reply:
agreed, good call